src/mkdocstrings/plugin.py

```diff
@@ -206,6 +207,9 @@ def on_post_build(self, config: Config, **kwargs) -> None:  # noqa: W0613,R0201
             log.debug("Tearing handlers down")
             self._handlers.teardown()
 
+            inv_contents = self._handlers.inventory.format_sphinx()
+            write_file(inv_contents, os.path.join(config["site_dir"], "objects.inv"))
```

Comment:
Wait,.. shouldn't adding this file be optional? For Crystal people will never use this but they will be annoyed by the file

Reply:
That's why I needed your review :smile: For the obvious things I missed! You're right, this should be made optional. Or we could generate it only if the Python handler was used.

Reply:
Ah so now I understand that adding these files by default is a good goal. I would suggest yeah, making it configurable, and enabled by default only for Python handler. But the config is still that all handlers produce inventory or none do, just that the default happens to be True only if there is at least 1 handler that asks for inventory. Kinda pseudo code:

```
try:
    use_inventory = self.config["enable_inventory"]
except KeyError:
    use_inventory = any(getattr(h, "wants_inventory", False) for h in handlers)
if use_inventory:
    write_file......
```